pages/linux/loadkeys.md

```diff
@@ -0,0 +1,28 @@
+# loadkeys
+
+> Load the kernel keymap for the console.
+> More information: <https://manned.org/loadkeys>.
+
+- Load a [d]efault keymap:
+
+`loadkeys --default`
+
+- Create a kernel source table [m]:
+
+`loadkeys --mktable`
+
+- Create a [b]inary keymap:
+
+`loadkeys --bkeymap`
+
+- Search and [p]arse keymap without action:
+
+`loadkeys --parse`
+
+- Suppress all normal output [q]:
```

Comment:
What this command does?
```suggestion
- Suppress all normal output:
```

Reply:
It will print to the standard output quietly without any details. We kind of say opposite of what _verbose_ option does.

Reply:
> What this command does?

It will print to the standard output quietly without any details. We kind of say opposite of what verbose option does.

Reply:
I'm not talking about the argument, I talk about the whole command. This command does the same that a argument-less example but without output to stdout. What the default behavior does?

Reply:
> I'm not talking about the argument, I talk about the whole command. This command does the same that a argument-less example but without output to stdout. What the default behavior does?

load the kernel keymap on console

Reply:
You mean we don't need to add this command specifically.